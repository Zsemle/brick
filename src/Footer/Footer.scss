@import '../style/layout.scss';
@import '../style/colors.scss';

.footer {
  padding: $space-small;
  background-color: $brick-dark-blue;
  color: $brick-white;

  @media (min-width: $bp-medium) {
    max-width: 920px;
    margin: auto;
  }
}
