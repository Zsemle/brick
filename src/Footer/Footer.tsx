import React, { FunctionComponent } from "react";

const Footer:FunctionComponent = () => {
    return (
        <header>
            <h1>Footer</h1>
        </header>
    )
}

export default Footer;