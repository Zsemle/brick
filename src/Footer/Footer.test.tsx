import { shallow, ShallowWrapper } from 'enzyme';
import React from 'react';
import Footer from './Footer';

describe('the footer component', () => {
  it('renders the component', () => {
    const wrapper:ShallowWrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });

});
