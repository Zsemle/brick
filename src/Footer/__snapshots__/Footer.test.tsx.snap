// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the footer component renders the component 1`] = `
<footer>
  <h1>
    Footer
  </h1>
</footer>
`;
